WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -171,7 +171,9 @@ final class MySiteViewController: UIViewController, UIScrollViewDelegate, NoSite
         subscribeToWillEnterForeground()
 
         if RemoteFeatureFlag.newGutenberg.enabled() {
-            GutenbergKit.EditorViewController.warmup()
+            GutenbergKit.EditorViewController.warmup(
+                configuration: blog.flatMap(EditorConfiguration.init(blog:)) ?? .default
+            )
```

Comment:
We can move/copy this code to "warm up" the GBK editor after switching sites, which pre-fetches the remote editor assets.